docs/development/using_omnisci.rst

```diff
@@ -18,8 +18,15 @@ or use it in your code:
    import modin.config as cfg
    cfg.StorageFormat.put('omnisci')
 
-Since OmniSci is run through its native engine Modin itself sets ``MODIN_ENGINE=Native``
-and you might not specify it explicitly.
+Since OmniSci is run through its native engine, Modin automatically sets ``MODIN_ENGINE=Native`` and you might not specify it explicitly.
+If for some reasons ``Native`` engine is explicitly set using modin.config or environment variable, make sure you also tell modin that
+``Experimental`` mode is turned on (``export MODIN_EXPERIMENTAL=true`` or ``cfg.IsExperimental.put(True)``) otherwise following error occurs:
```

Comment:
```suggestion
``Experimental`` mode is turned on (``export MODIN_EXPERIMENTAL=true`` or 
``cfg.IsExperimental.put(True)``) otherwise the following error occurs:
```